db/migrate/20190711074433_change_course_accrediting_provider_id_to_code.rb

```diff
@@ -0,0 +1,44 @@
+class ChangeCourseAccreditingProviderIdToCode < ActiveRecord::Migration[5.2]
+  class Course < ApplicationRecord
+    belongs_to :accrediting_provider, class_name: 'Provider', optional: true
+  end
+
+  def change
+    add_column :course, :accrediting_provider_code, :text
+    add_index :course, :accrediting_provider_code
+
+    reversible do |dir|
+      say_with_time 'updating accrediting provider on courses' do
+        Course.all.each do |course|
+          dir.up {
+            course.update accrediting_provider_code:
+                            course.accrediting_provider&.provider_code
+          }
+          # manage_courses_backend_development=# select count(*) from course where accrediting_provider_id is null;
```

Comment:
why 

 select count(*) from course where accrediting_provider_id `is null`;

same below